.github/workflows/cron-backup-pins.yaml

```diff
@@ -0,0 +1,40 @@
+name: Cron Backup Pins
+
+on:
+  schedule:
+    - cron: '*/30 * * * *'
```

Comment:
Why not just schedule for the max amount of time a job can run for 6h?

Reply:
If I understand this correctly the job has 2 goals:
1. move historical pins to EIPFS
2. keep moving new psa requests to EIPFS until we move to pickup.

For **2** I guess it's ideal to keep moving stuff as promptly as possible (30 min make sense, even less than that?) while we know the first runs of the job will be super slow (since they will have to go through all the historical data).

Isn't a solution to satisfy both words to keep the `schedule` as is and set `concurrency` on the job?